installers/nersc-sirepo-update/radiasoft-download.sh

```diff
@@ -44,3 +42,19 @@ nersc_sirepo_update_main() {
     chmod 711 ~
     chmod -R a+rX ~/.pyenv
 }
+
+nersc_sirepo_update_pyenv() {
+    virtualenv_name=$1
+    if ! pyenv versions --bare | grep -q "^$install_version_python$"; then
+        nersc_pyenv_no_global=1 install_repo_eval nersc-pyenv
+    fi
+    if [[ -e ~/.pyenv/versions/$virtualenv_name ]]; then
+        install_not_strict_cmd pyenv shell "$virtualenv_name"
+        if [[ $(python --version | cut -d' ' -f2) != $install_version_python ]]; then
```

Comment:
We should really add `nth` or equivalent to install, e.g. `$(install_nth 2 "$(python --version)")`, which abstracts the above, and doesn't use pipes.